src/assets/dropdown.js

```diff
@@ -0,0 +1,10 @@
+function test(){
+    let profileDropDownList = document.querySelector(".profile-dropdown-list");
+    let btn = document.querySelector(".profile-dropdown-btn");
+    profileDropDownList.classList.toggle('active');
+    window.addEventListener("click", function(e) {
+        if(!btn.contains(e.target)) {
+            profileDropDownList.classList.remove("active");
+        }
+    });
+}
```

Comment:
@Harshit8-Singh  why are we keeping it here in assets? let's try keep the logic in ts file only , if it is a test, let's just keep it in spec.ts file, we can do this in more angularly way, using eventlisteners and typescript